libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -2334,3 +2334,38 @@ string YulUtilFunctions::extractReturndataFunction()
 	});
 }
 
+string YulUtilFunctions::copyConstructorArgumentsToMemoryFunction(
+	ContractDefinition const& _contract,
+	string const& _creationObjectName
+)
+{
+	string functionName = "copy_arguments_for_constructor_" +
+		toString(_contract.constructor()->id()) +
+		"_object_" +
+		_contract.name() +
```

Comment:
Please add a `_` - the name of the contract could end in a digit.

Reply:
Ah, yes... Added it now